packages/tools/pie-css/README.md

```diff
@@ -121,15 +122,17 @@ Of course, you could include the styles straight inside your HTML document – m
 
 
 
-## Using the PIE CSS – SCSS helpers
+## Using pie-css for building web components
+> [!NOTE]
+> The following section is only relevant if you are building web components as a part of our component library within the PIE monorepo.
```

Comment:
QQ – I don't think this is necessarily true. Other applications could choose to use these SCSS helpers if they wanted to in their own applications.

Chatting to teams that use the CSS variables for things like font-sizes for instance, they have also had to create functions/mixins to convert the unitless values to have px suffixes.

I'd probably update this to just indicate that these helpers are used in the PIE web components and can optionally be used by consuming applications.

Reply:
I'll pick this up and raise a PR for this update